combat-trainer.lic

```diff
@@ -1211,6 +1223,11 @@ class TrainerProcess
       bput('meditate research turtle', 'You clear ')
     when 'Barb Research Debilitation'
       bput('meditate research earthquake', 'You clear ')
+    when 'Collect'
+      retreat
+      collect('rock')
```

Comment:
Does `collect` correctly handle the case in which your room won't let you collect rocks?